charters/Architecture/charter.md

```diff
@@ -0,0 +1,52 @@
+# DCP Architecture Team
+
+## Description
+
+The Data Coordination Platform (DCP) Architecture Team is responsible for the technical oversight of the DCP, including
+identifying and addressing *cross-cutting* architecture, integration, and operational concerns across DCP
+projects.
+
+The core team is composed of *Technical Leads* representing each technical DCP project group to ensure a diversity of
+skills and views needed for community technical decision-making. 
+
+## In-scope
+
+Software architecture responsibilities
+* Gather and synthesize DCP's end-to-end requirements
+* Set overall direction of architectural and software design direction for the DCP
+* Provide technical input for the PM product roadmap process
+* Review and provide final approval for technical charters for DCP projects
```

Comment:
I'm wondering if the [PM group](https://github.com/HumanCellAtlas/dcp-community/pull/23) should approve the User Stories section of these?  I'm worried that we have a formal way to connect the technical development of DCP with the user needs.

Reply:
It's not enough that technical charters go through an open round of review with PM before going to Tech Arch for approval?